commerce_coordinator/apps/core/tests/test_memcache.py

```diff
@@ -0,0 +1,122 @@
+"""
+Tests for memcache in util app
+"""
+
+
+from django.core.cache import caches
+from django.test import TestCase
+
+from commerce_coordinator.apps.core.memcache import safe_key
+
+
+class MemcacheTest(TestCase):
+    """
+    Test memcache key cleanup
+    """
+
+    # Test whitespace, control characters, and some non-ASCII UTF-16
+    UNICODE_CHAR_CODES = (list(range(30)) + [127] +
```

Comment:
this might be good to add to our testing utilities if we end up using it in more than one place.

Reply:
memcache.py and test_memcache.py is copied from LMS. My be we should move these files to edx-django-utils.